app/forms/csv_import_form.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+class CsvImportForm
+  include ActiveModel::Model
+  include ActiveModel::Validations
+  include ActiveModel::Attributes
+
+  attribute :user
+  attribute :file
+
+  validates_with CsvValidator
+
+  def initialize(attributes = {})
+    super(attributes)
```

Comment:
not sure we can remove the `initialize(attributes = {})` here or not as we just call `super` here 🤔

Reply:
No we cannot when I tried to do something like this
```ruby
def initialize
  super
```

this error will raise
```ruby
ArgumentError:
  unknown keyword: :user
```

Reply:
I think we cannot when I tried to do something like this
```ruby
def initialize
  super
```

this error will raise
```ruby
ArgumentError:
  unknown keyword: :user
```

Reply:
Oh hehe, I don't remember it much, good to know that :)